apps/kvittering-frontend/src/components/receipt-form.tsx

```diff
@@ -227,6 +227,7 @@ export default function ReceiptForm() {
 					url,
 					mime_type: files?.find((file) => file.url === url)?.file.type ?? "",
 				})),
+				// biome-ignore lint: Will never be undefined
```

Comment:
Would prefer if you specify which lint rule is broken instead of disabling all linting on the line

Reply:
Nice tip, I'll remember for later!